packages/recipe/src/components/EzCard/EzCardHeading.tsx

```diff
@@ -46,10 +46,15 @@ const EzCardHeading: FC<HeadingProps> = ({
   titleIcon,
   truncateHeading,
 }) => {
+  const accessibileTitle = `
+    ${title}
+    ${subtitle}
+  `;
+
   const heading = title && (
     <EzLayout layout="basic" alignY="top">
       {titleIcon}
-      <EzHeading size="3" subheading={subtitle}>
+      <EzHeading size="3" subheading={subtitle} title={truncateHeading ? accessibileTitle : null}>
```

Comment:
A minor note, but best practice is to use `undefined` here which means "do not use this attribute". See [definitions](https://exploringjs.com/impatient-js/ch_undefined-null.html#undefined-vs.-null).

```suggestion
      <EzHeading size="3" subheading={subtitle} title={truncateHeading ? accessibileTitle : undefined}>
```